_topic_map.yml

```diff
@@ -1166,12 +1156,6 @@ Topics:
 #### VIRTUAL MACHINE CHESS SALAD (silly name to highlight that the commented out assemblies need to be checked against merged filenams)
```

Comment:
Is the "Container-native virtualization user's guide" folder still helpful? I think it might be more helpful to move all of its contents directly into the CNV "bucket" and set up other logical groupings under the main header. I'd try to keep only the assemblies that have no similar assemblies at the top level, something like:

Container-native virtualization
- Release notes
> Installation
> Virtual machines
     > Creating virtual machines
     > Virtual machine networking
     ....
> Live migration
> Node maintenance
....
- Using the CLI tools

Reply:
That is an inspired suggestion. I created #18366 as a branch off this one to see how they compare and I think it's a great improvement over this one.